apps/sms/index.html

```diff
@@ -113,6 +113,7 @@
     <script defer src="js/contact_renderer.js"></script>
     <script defer src="js/information.js"></script>
     -->
+    <link rel="import" href="waiting-screen.html">
```

Comment:
also the build system actually breaks "rel=import" links... see https://bugzilla.mozilla.org/show_bug.cgi?id=1099185 I filed some time ago.